capstone/src/main/java/com/google/univiz/scorecard/CollegeIdReaderProviderImpl.java

```diff
@@ -0,0 +1,66 @@
+package com.google.univiz.scorecard;
+
+import static java.util.stream.Collectors.joining;
+
+import com.google.univiz.api.CollegeId;
+import com.google.univiz.config.UnivizConfig;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.Reader;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import javax.inject.Inject;
+
+class CollegeIdReaderProviderImpl implements CollegeIdReaderProvider {
+  private final UnivizConfig univizConfig;
+
+  @Inject
+  protected CollegeIdReaderProviderImpl(UnivizConfig univizConfig) {
+    this.univizConfig = univizConfig;
+  }
+
+  @Override
+  public Reader getReaderFromCollegeIds(List<CollegeId> ids) throws IOException {
+    Set<String> fields = new HashSet<>();
+    fields.add("id");
+    fields.add("school.name");
+    fields.add("school.city");
+    fields.add("school.main_campus");
+    fields.add("location.lat");
+    fields.add("location.lon");
+    fields.add("school.carnegie_size_setting");
+    fields.add("latest.admissions.admission_rate_overall");
+    fields.add("latest.admissions.sat_scores.average.overall");
+    fields.add("latest.student.size");
+    fields.add("latest.cost.attendance.academic_year");
+    fields.add("latest.student.demographics.men");
+    fields.add("latest.student.demographics.women");
```

Comment:
I think this addresses [your earlier question](https://github.com/googleinterns/univiz/pull/82#discussion_r470915661) then. Yea this would be particularly useful. I'll integrate this in.